packages/foam-vscode/test/utils.test.ts

```diff
@@ -1,10 +1,49 @@
 // @note: This will fail due to utils importing 'vscode'
 // which needs to be mocked in the jest test environment.
 // See: https://github.com/microsoft/vscode-test/issues/37
-import { dropExtension } from '../src/utils';
+import { dropExtension, removeBrackets } from '../src/utils';
 
 describe("dropExtension", () => {
   test("returns file name without extension", () => {
     expect(dropExtension('file.md')).toEqual('file');
   });
 });
+
+describe("removeBrackets", () => {
+  it("removes the brackets", () => {
+    const input = "hello world [[this-is-it]]";
+    const actual = removeBrackets(input);
+    const expected = "hello world This Is It";
```

Comment:
Perhaps it isn't for this PR, but I sometimes name files differently to my heading, so I would want to grab the title. Is that an easy change? No worries if not but just floating it...
Like the tests though!

Reply:
That's actually a very good point, especially if we already have the definitions. And will probably leverage more the foam metadata as opposed to simply the markdown itself

Reply:
Good point! I didn't even think about. 

>  Is that an easy change?

I don't know 😅 More than happy to accept change if one of y'all want to do it!